### DISCUSSION TITLE
Micrometer M2 version breaks build

### DISCUSSION THREAD

**Discussion by tomazfernandes** [spring-projects/spring-kafka#2121] [General]
Just a heads up, the Micrometer version that was recently updated to M2 changes packages around and at least one contract, so the current code in Main branch is not building.

Probably all PRs I'm opening will fail because of this, but I made sure to run the build them locally with the M1 version and they seem to be ok.

**Comment by tomazfernandes** [comment]
Wow, that was fast, thanks @garyrussell

**Reply by garyrussell** [reply]
Hehe - coincidence - I was given a heads up a little earlier - https://github.com/spring-projects/spring-kafka/issues/2068#issuecomment-1046303126

**Reply by tomazfernandes** [reply]
Oh, I see 😄 Thanks for looking into this on a Sunday anyway!

**Reply by garyrussell** [reply]
I have had to revert it because Boot is not planning on moving to that milestone any time soon.

https://github.com/spring-projects/spring-kafka/commit/b198c5b78b1117fd242b9efa601ca26fd7c4d1a1

**Reply by garyrussell** [reply]
Micrometer has now reverted the package changes.
